05week/spaceTravelToMars.js

```diff
@@ -9,7 +9,30 @@ let jobTypes = {
   programmer: 'Any Ship!'
 };
 
-// Your code here
+class CrewMember {
+  constructor (name, job, specialSkill, ship) {
+    this.name = name;
+    this.job = job;
+    this.specialSkill = specialSkill;
+  }
+    enterShip(ship) {
+    this.ship = ship;
+    ship.crew.push(this);
+  }
+}
+
+
+class Ship {
+  constructor (name, type, ability, crew) {
```

Comment:
Since you literally define `crew` as an empty array, you don't need to include this parameter in the constructor.

Reply:
ooh.. I see. I made the changes.